src/test/java/com/contrastsecurity/ContrastSDKTest.java

```diff
@@ -120,5 +119,19 @@ public void negativeTimeoutsAreNotSetTest() throws IOException {
     	assertNotEquals(connectionTimeout, conn.getConnectTimeout());
     	assertNotEquals(readTimeout, conn.getReadTimeout());
     }
+
+    @Test
+    public void testEnsureApi() throws IOException {
+        final String expectedApi = "http://localhost:19080/Contrast/api";
+        final String ensureUrlOne = ContrastSDKUtils.ensureApi("http://localhost:19080/Contrast/");
+        final String ensureUrlTwo = ContrastSDKUtils.ensureApi("http://localhost:19080/Contrast");
+
+        assertEquals(ensureUrlOne, expectedApi);
+        assertEquals(ensureUrlTwo, expectedApi);
+
+        final String ensureUrlIncorrect = ContrastSDKUtils.ensureApi("http://localhost:19080/");
+        assertNotEquals(ensureUrlIncorrect, expectedApi);

```

Comment:
Change this assertion to assert that the string is unchanged. Our method changes the string if in ends in Contrast, but if it doesn't end in Contrast, then the string should be returned unchanged.